app/src/main/java/crux/bphc/cms/core/FileExtensions.kt

```diff
@@ -0,0 +1,34 @@
+package crux.bphc.cms.core
+
+import android.content.Context
+import android.text.format.DateUtils
+import android.text.format.Formatter
+import crux.bphc.cms.R
+import crux.bphc.cms.utils.FileUtils
+import java.io.File
+import java.util.*
+
+fun File.getFormattedFileSize(context: Context) = Formatter.formatFileSize(context, this.getContentSize()).toUpperCase(Locale.ROOT)
+
+fun File.getContentSize(): Long {
+    return if (isDirectory) walkBottomUp().fold(0L, { res, it ->
+        res + it.length()
+    }) else length()
+}
+
+fun File.getDescription(context: Context) = if (isDirectory) {
+    when (listFiles()?.size ?: 0) {
+        0 -> context.getString(R.string.folder_no_items_zero)
+        1 -> context.getString(R.string.folder_no_items_one, getFormattedFileSize(context))
+        else -> context.getString(R.string.folder_no_items_many, listFiles()?.size
+                ?: 0, getFormattedFileSize(context))
```

Comment:
Place the `?:` operator before the line break. Or better yet, put the entire argument expression on the same line.